infra/base-images/base-builder-swift/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Copyright 2021 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/asra-ali/base-builder-new
```

Comment:
I think you meant to change this to gcr.io/oss-fuzz-base/base-builder-new before submitting the PR right?

Reply:
haha, yes oops I was locally testing. changing, so maybe CI can work after the images are pushed